docs/publishing-your-site.md

```diff
@@ -142,6 +142,14 @@ the following command from the directory containing the `mkdocs.yml` file:
 mkdocs gh-deploy --force
 ```
 
+This will build your documentation and deploy it to a branch
+`gh-pages` in your repository. See [this overview in the MkDocs
```

Comment:
"See this section" is not a good link name from an A11y point of view. We try to minimize those, because when you step with <kbd>tab</kbd> through the docs, your screenreander will not provide enough context. We definitely have some parts where we also do it like that, but when updating our documentation we should always try to give links discernible names, so that they can be understood on their own.

Reply:
Hope the wording is better now. Also tried to avoid the repetition of "MkDocs documentation".